functests/10-test-template-validator-placement.sh

```diff
@@ -48,6 +52,69 @@ if [ "$AFFINITY" != "testKey" ]; then
   RET=1
 fi
 
-oc delete -n ${TEST_NS} -f "${SCRIPTPATH}/10-template-validator-unversioned-cr.yaml" || exit 2
+oc delete -n ${TEST_NS} -f "${RES_DIR}/10-template-validator-affinity-nodeSelector-tolerations.yaml" || exit 2
+
+# Test if empty affinity/nodeSelector/tolerations values are propagated to the deployment
+oc create -n ${TEST_NS} -f "${RES_DIR}/10-template-validator-empty-affinity-nodeSelector-tolerations.yaml" || exit 2
```

Comment:
I think it would be better to move this line after the loop, that waits for deployment delete. This way, it is probably a race condition.

Reply:
Great catch, I agree